napari/layers/utils/_link_layers.py

```diff
@@ -207,7 +220,7 @@ def _get_common_evented_attributes(
         A set of layers to evaluate for attribute linking.
     exclude : set, optional
         Layer attributes that make no sense to link, or may error on changing.
-        {'thumbnail', 'status', 'name', 'data'}
+        {'thumbnail', 'status', 'name', 'mode', 'data', 'features', 'properties', 'extent', 'loaded'}
```

Comment:
Optional: I'd consider just removing this from the docstring because

- it's just repeating the default value, which is close in code and will be shown in rendered docs
- the value here is not consistent with that default value (e.g. `edge_color` is not present here

But this is optional here because this problem existed before this PR.